modules/system/holoportos/udiskie.nix

```
@@ -0,0 +1,40 @@
+{ config, pkgs, ... }:
+let
+  wrapSystemdScript = (
+    program: ''
+      source ${config.system.build.setEnvironment}
+      exec ${program}
+    ''
+  );
+in rec {
```

NixOS module definitions are never recursive sets, also see indentation:

```suggestion
in

{
```